caput/memh5.py

```diff
@@ -2588,6 +2598,9 @@ def _copy_from_file(h5group, memgroup, selections=None):
 
                     cdata = comm.bcast(cdata, root=0)
 
+                    # only needed until fixed: https://github.com/mpi4py/mpi4py/issues/177
+                    cdata = ensure_native_byteorder(pdata)
```

Comment:
This should be `cdata` as the argument, and it should also probably be before the `bcast` call. Obviously it's `Bcast` that's been causing us trouble, but just in case things change, or we change `bcast -> Bcast` at somepoint.

Reply:
Good catch on both counts! My eyes are getting tired.